### DISCUSSION TITLE
是不是能实现音乐视觉的效果

### DISCUSSION THREAD

**Discussion by jssda** [go-musicfox/go-musicfox#220] [Ideas]
能不能实现音乐视觉效果  在终端上   那可真是太酷了

![line](https://img-blog.csdnimg.cn/img_convert/54735c1e44de653119a7a53df900be77.png)
![脉动](https://img-blog.csdnimg.cn/img_convert/dfaedc284a8c2ec098b41517432acfe1.gif)

**Comment by anhoder** [comment]
可以直接用 https://github.com/karlstav/cava
![20240411_123440](https://github.com/go-musicfox/go-musicfox/assets/27605589/ff9914e2-299b-4241-9a85-401e7603a91a)
